libs/src/oracle2/services/assertions/gql/queries.ts

```diff
@@ -0,0 +1,44 @@
+import { chainsById } from "@shared/constants";
+import type { ChainId, OOV3GraphQuery } from "@shared/types";
+import { makeQueryName } from "@shared/utils";
+import request, { gql } from "graphql-request";
+
+export async function getAssertions(url: string, chainId: ChainId) {
+  const chainName = chainsById[chainId];
+  const queryName = makeQueryName("Optimistic Oracle V3", chainName);
+  const query = gql`
+    query ${queryName} {
```

Comment:
giving queries a name like this doesn't change the logic — it just allows us to intercept each of the queries by name with mock service worker